qiita_pet/handlers/api_proxy/tests/test_sample_template.py

```diff
@@ -65,6 +65,8 @@ def tearDown(self):
         if exists(self.new_study_fp):
             remove(self.new_study_fp)
 
+        qdb.study.Study.delete(self.new_study.id)
```

Comment:
@antgonza is the problem here that the tests fail because the study "Some New Study" already exists?